scripts/README.md

```diff
@@ -14,46 +14,87 @@ npx truffle compile
+Then continue the script and send out the second request, which will fund the bracket-traders' accounts on the exchange. Notice: One has to wait until the transaction is mined, as otherwise the nonce for the next request will be incorrect.
+Making the requests to the gnosis-interfaces does not cost any gas. However, signing and executing the transactions in the gnosis-safe interface will incur gas costs.
+
+Here is an example script invocation:
+
+```js
+npx truffle exec scripts/complete_liquidity_provision.js --targetToken=1 --stableToken=4 --lowestLimit=150 --hig
+hestLimit=200 --currentPrice=175 --masterSafe=$MASTER_SAFE --investmentTargetToken=10 --investmentStableToken=1000 --fleetSize=20 --network=$NETWORK_NAME
+```
+
+The prices must be specified in terms of 1 target token = x stable tokens.
+
+This example deploys a liquidity strategy with 20 brackets between the prices 150-200 on the pair WETH-USDC.
+In this script the targetToken is 1, which happens to be WETH and the stableToken is 4, which happens to be USDC. The token ids of the exchange contract can be read from Etherscan.info in the 'Contract/Read Contract' tab, e.g. [here for mainnet](https://etherscan.io/address/0x6f400810b62df8e13fded51be75ff5393eaa841f)
 
-These script require to set up a "Master Safe" created through the interface (here)[https://rinkeby.gnosis-safe.io] with an owner key under your control. This Master Safe should have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface and this implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
-In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. Otherwise, everyone can steal the funds from your account!
+The fleet size should be smaller than or equal to 20, in order to ensure that the transactions can be send via Metamask - otherwise, it can happen that the payload is too high for Metamask.
 
-For the signing process, note that Metamask underestimates gas consumption. Just make sure that the gas limit of your transaction is sufficient. There is currently no proper way to estimate the gas limit. In practice, setting the gas limit to a value higher than 5,000,000 for transactions involving 20 brackets is a practical, yet not fully reliable way to ensure a successful transaction.
+Please document the displayed bracket-trader addresses. They are require for the withdrawals later. They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
```

Comment:
```suggestion
Please document the displayed bracket-trader addresses. They are required for future withdrawals. They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
```